service/rpc/servicer.go

```diff
@@ -419,6 +419,320 @@ func (s *Servicer) BlockTransaction(ctx context.Context, request *types.BlockTra
 	}, nil
 }
 
+func (s *Servicer) Call(
+	ctx context.Context,
+	callRequest *types.CallRequest,
+) (*types.CallResponse, *types.Error, error) {
+	switch callRequest.Method {
```

Comment:
i would suggest using a map of `allowedMethod => handlerFunction` instead of a switch here. To make it more flexible and readable